enos/modules/upgrade_client/scripts/verify_allocs.sh

```diff
@@ -48,17 +48,50 @@ done
 
 echo "Client $client_id at $CLIENT_IP is ready"
 
-# Quality: "nomad_alloc_reconect: A GET call to /v1/allocs will return the same IDs for running allocs before and after a client upgrade on each client"
-echo "Allocs found before upgrade $ALLOCS"
+allocs_count=$($ALLOCS |jq '[ .[] | select(.ClientStatus == "running")] | length')
+
+# Quality: "nomad_alloc_reconnect: A GET call to /v1/allocs will return the same IDs for running allocs before and after a client upgrade on each client"
+echo "$allocs_count allocs found before upgrade $ALLOCS"
+
+checkAllocsCount() {
+    local allocs
+    allocs=$(nomad alloc status -json) || error_exit "Failed to check alloc status"
```

Comment:
Shouldn't we filter this down to the node we're looking for? The `ALLOCS` env var we're passing to this script does.

Reply:
good point, I copied it and forgot to add the flag